### ISSUE TITLE
Replace 'message' with 'detail'

### ISSUE THREAD

**Issue by priyanshisharma** [ECellNitrr/teesco-backend#113]
**Is your feature request related to a problem? Please describe.**
It maybe observed that `"message"` and `"detail"` has been used interchangeably as keys for Responses in `org.responses.py`, this needs to be ,made uniform. Such that, "detail" is used instead.

**Describe the solution you'd like**
Go to `org.responses.py`, and replace all instances of `message` with `detail`.

For example:
```
user_unauthorized_403 = {
    "message":"You do not have the required permissions."
}
```
becomes
```
user_unauthorized_403 = {
    "detail":"You do not have the required permissions."
}
```

**Comment by adityatiwari23** [comment]
I want to fix the issue. Please let me give a try, as I am beginner, want to start with your project.
I hope presently no one is working on this issue

**Comment by sahil9001** [comment]
There you go, @Adityatiwar1 !

**Comment by naveennvrgup** [comment]
This issue is solved in #126 but the issue is not linked in the PR properly. So I am closing this issue.
